src/components/receipt-view/styles/data-breakdown-aggregate.module.css

```diff
@@ -0,0 +1,8 @@
+.breakdownItem {
+  min-height: 10vh;
+  text-align: center;
+}
+
+.breakdownBox {
+  padding: 3em 1em;
+}
```

Comment:
Are your prettier and eslint set up correctly? I notice a few files missing newlines at the end